dora/core/server/master/src/main/java/alluxio/master/job/DoraLoadJob.java

```diff
@@ -109,9 +108,8 @@ public class DoraLoadJob extends AbstractJob<DoraLoadJob.DoraLoadTask> {
   private boolean mVerificationEnabled;
 
   // Job states
-  private final LinkedList<String> mRetryFiles = new LinkedList<>();
+  private final Queue<UfsStatus> mRetryFiles = new ArrayDeque<>();
```

Comment:
i'm not convinced that linked list should be used here

Reply:
also store the ufs status itself instead of the path string so that we can avoid calling getStatus() again upon failure